ChangeLog

```diff
@@ -205,6 +201,10 @@ Release date: TBA
 
   Closes pylint-dev/pylint#8749
 
+* Fix a crash when inferring a `typing.TypeVar` call.
+
+  Closes pylint-dev/pylint#8802
+
```

Comment:
Actually 2.15.6 has since been released since my comment, so we need a new header for 2.15.7 (and to add the release date for 2.15.6)

Reply:
Also it takes double backticks for code literals.